site/fetchers/blog-posts/test.js

```diff
@@ -30,7 +30,7 @@ describe('featured blog posts fetcher', () => {
   it('maps blog posts to expected data shape', () => {
     const data = mapDataToState(fixture());
     const post = data[0];
-    expect(post).to.have.all.keys('slug', 'category', 'title', 'author');
+    expect(post).to.have.all.keys('slug', 'category', 'title', 'author', 'date', 'excerpt');
```

Comment:
Should we test the values as well? I'd suggest to use [toMatchObject](http://facebook.github.io/jest/docs/en/expect.html#tomatchobjectobject) here.